buildbot/osuosl/master/config/workers.py

```diff
@@ -41,7 +41,6 @@ def get_all():
         create_worker("linaro-g3-04", max_builds=1),
 
         # AArch64 Windows Microsoft Surface X Pro
-        create_worker("linaro-armv8-windows-msvc-01", max_builds=1),
```

Comment:
Be careful with the timing of when this change lands vs. when you remove the agent from the actual machine. I managed to get our server IP banned because I left some workers running that kept trying to connect to the build master.

If the machine is offline before this lands there should be no problem.